features/api_security.feature

```diff
@@ -29,7 +29,16 @@ Feature: API security/security status tests
         """
         When I run `apt update` with sudo
         And I run `apt upgrade -y` with sudo
-        And I run `apt install jq bzip2 libopenscap8 -y` with sudo
+        And I run `apt install jq bzip2 -y` with sudo
+        # Install the oscap version 1.3.7 which solved the epoch error message issue
+        And I run `apt-get install lcov swig xsltproc rpm-common lua5.3 libyaml-dev libapt-pkg-dev libdbus-1-dev libdbus-glib-1-dev libcurl4-openssl-dev libgcrypt-dev libselinux1-dev libgconf2-dev libacl1-dev libblkid-dev libcap-dev libxml2-dev libxslt1-dev libxml-parser-perl libxml-xpath-perl libperl-dev librpm-dev librtmp-dev libxmlsec1-dev libxmlsec1-openssl cmake make build-essential -y` with sudo
```

Comment:
This seems to have some extra packages. I think all we need is specified here: (or is this documentation out of date?) https://github.com/OpenSCAP/openscap/blob/maint-1.3/docs/developer/developer.adoc

They say:
```
sudo apt-get install -y cmake libdbus-1-dev libdbus-glib-1-dev libcurl4-openssl-dev \
libgcrypt20-dev libselinux1-dev libxslt1-dev libgconf2-dev libacl1-dev libblkid-dev \
libcap-dev libxml2-dev libldap2-dev libpcre3-dev python-dev swig libxml-parser-perl \
libxml-xpath-perl libperl-dev libbz2-dev librpm-dev g++ libapt-pkg-dev libyaml-dev \
libxmlsec1-dev libxmlsec1-openssl
```

Reply:
Ah, I did not see the docs, I just went straight to the github actions. Thanks for this.